nightly-metrics-update.sh

```diff
@@ -34,13 +34,28 @@ cd ../osg-cpu-hours
 
 cd ..
 
+# Clean the environment
+deactivate
+
+# Run the project waittime
+pushd osg-project-waittime
+python3 -m venv venv
+. venv/bin/activate
+pip install -r requirements.txt
+./calculate-waittime.py $START_DATE $END_DATE
+
+# outputs as output.csv
+cp output.csv osg-waittime.csv
```

Comment:
```suggestion
cp output.csv ../osg-waittime.csv
```

If i follow right, you are in the `osg-project-waittime` dir here, but you want the output file to go to the main parent dir.  (See the other metrics run above for examples.)